celery_worker/worker.py

```
@@ -3,110 +3,183 @@
+                        "role": "assistant",
+                        "text": answer,
+                        "content": answer,
+                        "ts": finish_time
+                    }
+                }
+
+        # Выполняем сохранение в MongoDB
+        db["drawings"].update_one({"id": drawing_id}, update_data)
+        notify_update(drawing_id, final_status, answer=answer)
 
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)